module/VuFind/src/VuFind/View/Helper/Root/Record.php

```diff
@@ -419,10 +419,10 @@ public function getSearchResult($view)
      */
     public function getCheckbox($idPrefix = '', $formAttr = false, $number = null)
     {
-        $id = $this->driver->getSourceIdentifier() . '|'
-            . $this->driver->getUniqueId();
-        $context
-            = ['id' => $id, 'number' => $number, 'prefix' => $idPrefix];
+        $context = compact('number') + [
```

Comment:
It looks to me like this refactoring is dropping the `prefix` element from the array. While I realize in the long-term we may no longer need that value, I think in the short-term, we probably still need it (unless there are corresponding template changes that are meant to be in this PR but are somehow missing).

Reply:
Interestingly, it does appear that prefix is unused by the current [checkbox.phtml](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/templates/record/checkbox.phtml), so maybe it is okay to remove it. But it might also be worth updating the template to actually include the ID value since we're going to the trouble of creating it. The only question is whether there's any risk of that causing duplicate element IDs under any circumstances.